api/requirements.txt

```diff
@@ -7,4 +7,4 @@ spacy==2.2.4
 tensorflow-gpu==2.1.0
 tensorflow-text==2.1.1
```

Comment:
We can remove these two now

Reply:
We still need TF for `from tensorflow.python.lib.io import file_io` -- maybe we can switch to the CPU version? I also briefly tried the Google Cloud storage Python library before, but it required credentials while the TF implementation allows anonymous accounts.

Reply:
I see.. but do we still need tensorflow-text? It was used by T5's Sentencepiece tokenizer, but I believe huggingface uses its own implementation.

Reply:
Nah, we don't need that.